src/services/request-service.ts

```diff
@@ -109,7 +109,7 @@ export class RequestService {
       did,
       schema: genesisFields?.schema,
       family: genesisFields?.family,
-      model: genesisFields?.model,
+      model: genesisFields?.model?.toString() ?? '',
```

Comment:
actually wait, I'm not sure this will actually log the right thing.  Have you tested this with a real MID stream's genesis commit?  I think you need to do `StreamID.fromBytes(genesisFields.model).toString()`

Reply:
I'm not 100% positive though, so if you tested it with just calling toString on the Uint8Array and it printed the proper model streamid string we expect, then it's fine.

Reply:
No, so the thing is, because we decode it actually model is an object.

so, the structure of 'model' depends on whether it was run thru decode or not

```


carFile.get(genesisJWS.link).header.model

Uint8Array(40)


decode(IpfsGenesis, carFile.get(genesisJWS.link)).header.model

{ _tag: Symbol(@ceramicnetwork/streamid/StreamID),

_type: 2,

_cid: CID }


```

and in our case, it is using decode

https://github.com/ceramicnetwork/ceramic-anchor-service/blob/057158e8ab0c735a4a7061dbcca82eaa3bb6e19b/src/services/metadata-service.ts#L98

```


const genesis = decode(IpfsGenesis, genesisRecord)
return genesis.header
```

Reply:
@stbrody fyi ^^ so i commited your change, will merge when it completes

Reply:
Ah I see, we use the codec to deserialize the genesis commit, so it's already a StreamID instance by the time we get here.  Sounds good.